ibtk/include/ibtk/ProblemSpecification.h

```diff
@@ -0,0 +1,120 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (c) 2011 - 2023 by the IBAMR developers
+// All rights reserved.
+//
+// This file is part of IBAMR.
+//
+// IBAMR is free software and is distributed under the 3-clause BSD
+// license. The full text of the license can be found in the file
+// COPYRIGHT at the top level directory of IBAMR.
+//
+// ---------------------------------------------------------------------
+
+/////////////////////////////// INCLUDE GUARD ////////////////////////////////
+
+#ifndef included_IBTK_ProblemSpecification
+#define included_IBTK_ProblemSpecification
+
+/////////////////////////////// INCLUDES /////////////////////////////////////
+
+#include <ibtk/config.h>
+
+#include "ibtk/ibtk_utilities.h"
+
+#include <limits>
+
+/////////////////////////////// FUNCTION DEFINITIONS /////////////////////////
+
+namespace IBTK
+{
+
+/*!
+ * @brief An empty structure holding problem specifications for operators and solvers.
+ *
+ */
+struct ProblemSpecification
+{
+};
+
+/*!
+ * \brief Class VCViscousOpSpec is a light-weight structure holding
+ * values of VCViscousOp coefficients.
+ */
+struct VCViscousOpSpec : public ProblemSpecification
+{
+    VCViscousOpSpec()
```

Comment:
Please put `inline` implementations in `include/ibtk/private/VCViscousOpSpec-inl.h`.